docs/release.md

```diff
@@ -0,0 +1,77 @@
+A "FIWARE release" contains versioned releases of all Generic Enablers found
+within the [catalogue](https://www.fiware.org/developers/catalogue/). All
+components must work seamlessly with the current context broker and each other.
+
+The following timetable is proposed for a typical FIWARE release cycle, where it
+is assumed that releases will follow a monthly cadence.
+
+## Release Prerequisites
+
+Additional GE development for each release cycle should close by the beginning
+of the month - ideally this should mean that all code to be included in the
+FIWARE release has been:
+
+-   Written
+-   Tested
+-   Documented
+-   Labelled as a [SemVer release](GE_Requirements.md#releases) on **GitHub**
+
+Any proposed [new entries](GE_Checklist.md#new-generic-enablers) to the
+catalogue **MUST** have been presented to the TSC
+
+Any component
+[transitioning to a full member](GE_Checklist.md#transition-to-full-membership)
+of the catalogue **MUST** have completed all "Must" requirements and have been
+presented as such to the TSC
+
+## Release timetable
+
+#### Week 1
+
+The formal commencement of the release process will be announced during the
+**first** TSC meeting of the month
+
+-   A list of release candidates will be collated for the TSC and created in a
+    prerelease candidate list on
+    [GitHub](https://github.com/Fiware/catalogue/releases)
+-   For each Generic Enabler
+    -   The latest tagged **GitHub** release is assumed to part of the current
+        release cycle unless explicitly informed otherwise in writing.
+    -   Where no release is found, the latest tag will be proposed.
+    -   Where no tags are found, the latest hash will be proposed.
+-   On this basis all GE Owners should immediately either:
+    -   Double tag their **existing** approved release version in their GitHub
+        repository with `FIWARE_<major>.<minor>`
```

Comment:
is it really necessary to double tag? perhaps there can be projects that just release for FIWARE.

Reply:
The wording could probably be improved here. I just want to ensure that the FIWARE release is somehow trackable within the GE code repos. I'm less bothered about the actual mechanism 

It still would be nice to have consistent double tagging though - it would make it much easier to create consistent builds which work nicely together - just pick the `FIWARE_7.4` for all components for example. 

-

Reply:
yes, double tagging is a good idea